src/ndarray/ndarray_function-inl.h

```diff
@@ -446,6 +468,11 @@ DECL_SCALAR(DEVICE, Minus, EvalScalar_, true)
 DECL_SCALAR(DEVICE, Mul, EvalScalar_, true)
 DECL_SCALAR(DEVICE, Div, EvalScalar_, true)
 
+DECL_BINARY_LAUNCH(DEVICE, Plus)
```

Comment:
why do you have both DECL_BINARY_LAUNCH and DECL_BINARY for Plus?

Reply:
Removed the earlier ones, thanks